packages/react-components/src/components/TextFieldWithMention/mentionTagUtils.test.ts

```diff
@@ -0,0 +1,387 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Mention } from '../MentionPopover';
+import {
+  TagData,
+  findMentionTagForSelection,
+  findNewSelectionIndexForMention,
+  findStringsDiffIndexes,
+  getDisplayNameForMentionSuggestion,
+  getValidatedIndexInRange,
+  htmlStringForMentionSuggestion,
+  rangeOfWordInSelection,
+  textToTagParser,
+  updateHTML
+} from './mentionTagUtils';
+
+import { COMPONENT_LOCALE_EN_US } from '../../localization/locales';
+
+describe('Mention logic should be robust and accurate', () => {
+  const localeStrings = COMPONENT_LOCALE_EN_US.strings;
+
+  const basicMention = 'Hello <msft-mention id="1" displayText="Everyone">everyone</msft-mention>!';
```

Comment:
Can the displayName be as an attr be different than the text wrapped up between the tags?

Reply:
Yes, it can be edited to be that way